README.rst

```diff
@@ -59,7 +59,7 @@ Features
   ``ISWAP``, ``PSWAP``, and ``CPHASE``. Every custom operation supports analytic
   differentiation.
```

Comment:
These gates are now defined in PennyLane:
- `ISWAP`: 
https://github.com/PennyLaneAI/pennylane/blob/e79f264adf743dbd35f2811de367edb9789718c3/pennylane/ops/qubit/non_parametric_ops.py#L1426

- `PSWAP`: https://github.com/PennyLaneAI/pennylane/blob/e79f264adf743dbd35f2811de367edb9789718c3/pennylane/ops/qubit/parametric_ops.py#L3189

- `CPHASE`: 
https://github.com/PennyLaneAI/pennylane/blob/e79f264adf743dbd35f2811de367edb9789718c3/pennylane/ops/qubit/parametric_ops.py#L546

Reply:
Should I just go ahead and remove this whole bullet point then?

Reply:
Sure!